### ISSUE TITLE
Crashing on export

### ISSUE THREAD

**Issue by goaldtesting** [LeGoffMael/video_editor#100]
Hi there, on iOS the app crashes when I try export and I'm getting the following output in the console? This is using the code from the example.
 stop reason = EXC_RESOURCE RESOURCE_TYPE_MEMORY (limit=3072 MB, unused=0x0)

**Comment by LeGoffMael** [comment]
Hi, no you should be able to export on debug mode as well.

It looks like the exportation takes too much resources on your phone.
You should probably be able to export a shorter video.

**Comment by gperez-88** [comment]
same issue on android OutofMemoryError in visualizing trimmer or during exporting

android 8.1
ram: 2gb

**Comment by AgentRachithHegde** [comment]
The same issue for me in Android it is working but in iOS, if the video is taken from the phone I can't export it because the app is crashing.

**Comment by LeGoffMael** [comment]
Is this still an issue in the latest version (since 2.0.0) ?

**Comment by AgentRachithHegde** [comment]
I will check it and confirm @LeGoffMael

**Comment by ParhaMDeF** [comment]
same problem with ios (not sure about android). the problem came when I upgraded flutter to the latest version.
This is the error:
`stop reason = EXC_RESOURCE RESOURCE_TYPE_MEMORY (limit=3072 MB, unused=0x0)
`
package version: 1.5.4

**Comment by ParhaMDeF** [comment]
I did some research. I think the problem is `video_player` package. check this link out.

https://github.com/flutter/flutter/issues/96370

**Comment by ParhaMDeF** [comment]
> The same issue for me in Android it is working but in iOS, if the video is taken from the phone I can't export it because the app is crashing.

I fixed the problem of exporting video taken by phone by setting the resolution to high or medium.
I used a `camera` package to record videos.
it may help.

**Comment by preteambuy** [comment]
is there a fix friends? We are waiting on this fix to release our app. When we export the video recorded using camera the app crashes.

**Comment by furkeen** [comment]
Is there any solution?

**Comment by LeGoffMael** [comment]
The latest version 3.0.0 is out.
The exportation process is not part of the package anymore.
You can now have full control over the exportation process.
